lib/ret/avatar_listing.ex

```diff
@@ -49,9 +49,11 @@ defmodule Ret.AvatarListing do
 
   def url(%AvatarListing{} = avatar), do: "#{RetWeb.Endpoint.url()}/api/v1/avatars/#{avatar.avatar_listing_sid}"
 
-  def gltf_url(%AvatarListing{} = avatar), do: "#{AvatarListing.url(avatar)}/avatar.gltf?v=#{AvatarListing.version(avatar)}"
+  def gltf_url(%AvatarListing{} = avatar),
```

Comment:
These utility functions are almost identical between `avatar` and `avatar_listing`. It would be better to adopt the type guard form that we use for scenes:
https://github.com/mozilla/reticulum/blob/d3578d862f7d7593ee3399c020ed8f53e7ca2c93/lib/ret/scene.ex#L43